src/locale/messages.fr.xlf

```diff
@@ -14,6 +14,9 @@
       <trans-unit id="name" datatype="html">
         <target>Nom</target>
       </trans-unit>
+       <trans-unit id="socialOrganizationName" datatype="html">
+          <target>Nom de l'organisation sociale</target>
```

Comment:
The purpose of this field is to have an article before the organization name if needed. It will be useful in notification contents. 

For better undestanding, you may change the translation with "Nom de l'organisation préfixée par un article (ex : La Société)"